app/controllers/helpers/SequenceIdHelper.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.helpers
+
+import models.DeclarationMeta
+import models.DeclarationMeta.sequenceIdPlaceholder
+import models.declaration.DeclarationStatus.preSubmissionStatuses
+import models.declaration.{EsoKeyProvider, ExplicitlySequencedObject}
+
+class SequenceIdHelper {
```

Comment:
Why a `class`, which you need to inject, instead f an `object` ?

Reply:
Since you are unit-testing the controllers, you could check the "final" product, rather the intermediate steps (the params passed to `SequenceIdHandler`), by using `verify` on the resulting declaration passed as parameter to the `ExportsCache` mock. That's the final goal of unit tests, isn't it?